client/test-data/corsica.locoflex.rococo.json

```diff
@@ -0,0 +1,120 @@
+{
+  "type": "FeatureCollection",
+  "community": {
+    "meta": {
+      "name": "Corsica LoCoFlex",
+      "symbol": "CLL",
+      "assets": "QmP2fzfikh7VqTu8pvzd2G2vAd4eK7EaazXTEgqGN6AWoD",
+      "announcementSigner": {
+        "bip340": "0xceda72c19f11aebecf8fb4a0ab3fb2a03e03f283225655baab4728303a560a13"
+      },
+      "rules": "loCoFlex"
+    },
+    "bootstrappers": [
+      "5FyWbcwN1TGPdyzRzoEeem3MUcc7jXRs7ZoftZkAQLV47nS7",
```

Comment:
I'd prefer to add all test accounts (like Adriana community) for flexibility, but I guess the community is already registered? Then we leave it.

Reply:
Yeah, I don't have the other Adriana accounts and its metadata is not in the repository, but I don't really see the benefit of having more than 3 here.

Reply:
Added them